packages/theming/theme-tokens/src/getAliasTokens.ts

```diff
@@ -0,0 +1,10 @@
+import * as lightAliasTokens from './light/tokens-alias';
+import * as darkAliasTokens from './dark/tokens-alias';
+import { AppearanceOptions } from '@fluentui-react-native/theme-types';
+
+export function getAliasTokens(mode: AppearanceOptions) {
+  if (mode === 'light') return lightAliasTokens.default;
+  else if (mode === 'dark') return darkAliasTokens.default;
+
+  return lightAliasTokens.default;
```

Comment:
I'm not sure if typescript will let us, but could we make this a switch statement without a default case so it _has_ to fill every case and therefore protect us against future AppearanceOption additions?

Reply:
I'll have to look into whether TS natively has something like "assertNever" (which basically does what you're asking for), I'll get back to you

Reply:
https://github.com/microsoft/fluentui-react-native/pull/799